config/routes.rb

```diff
@@ -20,10 +22,11 @@
   scope module: 'dashboard' do
     resources :user_sessions, only: :destroy
     resources :users, only: :destroy
-    post 'logout' => 'user_sessions#destroy', :as => :logout
+    post 'logout' => 'user_sessions#destroy', as: :logout
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.